src/libraries/Microsoft.PowerFx.Core/Public/Values/FormulaValue.cs

```diff
@@ -157,6 +157,22 @@ public static UntypedObjectValue New(IUntypedObject untypedObject)
                 untypedObject);
         }
 
+/*
+        // $$$
```

Comment:
>// $$$ [](http://example.com/codeflow?start=8&length=6)

We should be able to cleanup a lot of New() code in this file and leverage the providers.